tests/base/quadrature_test.output

```diff
@@ -94,22 +94,22 @@ DEAL:2d:faces::Quadrature no.17 is exact for polynomials of degree 5
 DEAL:2d:faces::Quadrature no.18 is exact for polynomials of degree 7
 DEAL:2d:faces::Quadrature no.19 is exact for polynomials of degree 9
 DEAL:2d:faces::Quadrature no.20 is exact for polynomials of degree 11
-DEAL:2d:faces::Quadrature no.21 is exact for polynomials of degree 0
-DEAL:2d:faces::Quadrature no.22 is exact for polynomials of degree 0
-DEAL:2d:faces::Quadrature no.23 is exact for polynomials of degree 2
-DEAL:2d:faces::Quadrature no.24 is exact for polynomials of degree 2
-DEAL:2d:faces::Quadrature no.25 is exact for polynomials of degree 4
-DEAL:2d:faces::Quadrature no.26 is exact for polynomials of degree 4
-DEAL:2d:faces::Quadrature no.27 is exact for polynomials of degree 6
-DEAL:2d:faces::Quadrature no.28 is exact for polynomials of degree 6
-DEAL:2d:faces::Quadrature no.29 is exact for polynomials of degree 8
-DEAL:2d:faces::Quadrature no.30 is exact for polynomials of degree 8
-DEAL:2d:faces::Quadrature no.31 is exact for polynomials of degree 10
-DEAL:2d:faces::Quadrature no.32 is exact for polynomials of degree 10
-DEAL:2d:faces::Quadrature no.33 is exact for polynomials of degree 12
-DEAL:2d:faces::Quadrature no.34 is exact for polynomials of degree 12
-DEAL:2d:faces::Quadrature no.35 is exact for polynomials of degree 14
-DEAL:2d:faces::Quadrature no.36 is exact for polynomials of degree 14
+DEAL:2d:faces::Quadrature no.21 is exact for polynomials of degree 1
+DEAL:2d:faces::Quadrature no.22 is exact for polynomials of degree 1
+DEAL:2d:faces::Quadrature no.23 is exact for polynomials of degree 3
+DEAL:2d:faces::Quadrature no.24 is exact for polynomials of degree 3
+DEAL:2d:faces::Quadrature no.25 is exact for polynomials of degree 5
+DEAL:2d:faces::Quadrature no.26 is exact for polynomials of degree 5
+DEAL:2d:faces::Quadrature no.27 is exact for polynomials of degree 7
+DEAL:2d:faces::Quadrature no.28 is exact for polynomials of degree 7
+DEAL:2d:faces::Quadrature no.29 is exact for polynomials of degree 9
+DEAL:2d:faces::Quadrature no.30 is exact for polynomials of degree 9
+DEAL:2d:faces::Quadrature no.31 is exact for polynomials of degree 11
+DEAL:2d:faces::Quadrature no.32 is exact for polynomials of degree 11
+DEAL:2d:faces::Quadrature no.33 is exact for polynomials of degree 13
+DEAL:2d:faces::Quadrature no.34 is exact for polynomials of degree 13
+DEAL:2d:faces::Quadrature no.35 is exact for polynomials of degree 15
+DEAL:2d:faces::Quadrature no.36 is exact for polynomials of degree 15
```

Comment:
What changed that made these quadrature formulas all one degree more exact?

Reply:
I didn't look into it that closely, but a quirk of that test is that it integrates using every face formula at once: e.g., in 3d it integrates the function 8 times since `QProjector` creates 8 quadratures per face (of a hex). Hence, we are now integrating it twice per face in 2d instead of once, so the (approximated) integral is different and, to match what is done in 3d, the error tolerance is different.